models/webhook.go

```diff
@@ -651,15 +651,9 @@ type HookTask struct {
 	RepoID          int64 `xorm:"INDEX"`
 	HookID          int64
 	UUID            string
-	Typ             HookTaskType `xorm:"VARCHAR(16) index"`
```

Comment:
Why remove this column?

Reply:
A HookTask is linked to a Webhook with the `HookID` and the Webhook knows which type it has. It was only used to determine if it is used for Matrix so no need to save it for every HookTask.